[2023-01-04T14:59:20.015+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: titanic_etl_pipeline.create_dashboard_task manual__2023-01-04T14:59:13.547919+00:00 [queued]>
[2023-01-04T14:59:20.033+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: titanic_etl_pipeline.create_dashboard_task manual__2023-01-04T14:59:13.547919+00:00 [queued]>
[2023-01-04T14:59:20.036+0000] {taskinstance.py:1283} INFO - 
--------------------------------------------------------------------------------
[2023-01-04T14:59:20.038+0000] {taskinstance.py:1284} INFO - Starting attempt 1 of 2
[2023-01-04T14:59:20.039+0000] {taskinstance.py:1285} INFO - 
--------------------------------------------------------------------------------
[2023-01-04T14:59:20.075+0000] {taskinstance.py:1304} INFO - Executing <Task(PythonOperator): create_dashboard_task> on 2023-01-04 14:59:13.547919+00:00
[2023-01-04T14:59:20.082+0000] {standard_task_runner.py:55} INFO - Started process 2740 to run task
[2023-01-04T14:59:20.088+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'titanic_etl_pipeline', 'create_dashboard_task', 'manual__2023-01-04T14:59:13.547919+00:00', '--job-id', '61', '--raw', '--subdir', 'DAGS_FOLDER/titanic_etl.py', '--cfg-path', '/tmp/tmpl2dt9hf7']
[2023-01-04T14:59:20.094+0000] {standard_task_runner.py:83} INFO - Job 61: Subtask create_dashboard_task
[2023-01-04T14:59:20.231+0000] {task_command.py:389} INFO - Running <TaskInstance: titanic_etl_pipeline.create_dashboard_task manual__2023-01-04T14:59:13.547919+00:00 [running]> on host 82367dbeb80f
[2023-01-04T14:59:20.342+0000] {taskinstance.py:1513} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=titanic_etl_pipeline
AIRFLOW_CTX_TASK_ID=create_dashboard_task
AIRFLOW_CTX_EXECUTION_DATE=2023-01-04T14:59:13.547919+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-01-04T14:59:13.547919+00:00
[2023-01-04T14:59:20.381+0000] {logging_mixin.py:137} INFO - Dash is running on http://0.0.0.0:8050/
[2023-01-04T14:59:20.381+0000] {dash.py:1968} INFO - Dash is running on http://0.0.0.0:8050/

[2023-01-04T14:59:20.391+0000] {logging_mixin.py:137} INFO -  * Serving Flask app '***'
[2023-01-04T14:59:20.392+0000] {logging_mixin.py:137} INFO -  * Debug mode: off
[2023-01-04T14:59:20.392+0000] {logging_mixin.py:137} WARNING - Address already in use
[2023-01-04T14:59:20.393+0000] {logging_mixin.py:137} WARNING - Port 8050 is in use by another program. Either identify and stop that program, or start the server with a different port.
[2023-01-04T14:59:20.423+0000] {local_task_job.py:159} INFO - Task exited with return code 1
[2023-01-04T14:59:20.451+0000] {taskinstance.py:2582} INFO - 0 downstream tasks scheduled from follow-on schedule check
